packages/core/src/index.ts

```diff
@@ -1,8 +1,16 @@
-export type { BundleOutputFormat } from './utils';
+export { BundleOutputFormat, readFileFromUrl } from './utils';
```

Comment:
@RomanHotsiy when you get a chance, could you check this change?

Reply:
lgtm